recipe/meta.yaml

```diff
@@ -37,7 +38,7 @@ test:
 
 about:
     home: https://github.com/silx-kit/fabio
-    license: GPL v3.0, GPL v2.0, MIT, GPL, LGPL v3.0, BSD 3-Clause, Apache 2.0
+    license: MIT, LGPL v3.0
```

Comment:
I think this can actually be provided as a yaml list

Reply:
Linting doesn't like it...

Reply:
hmm, ok I guess we need to fix the linting at some point